src/assets/scripts/client/navigationLibrary/StandardRoute/StandardRouteModel.js

```diff
@@ -291,6 +291,25 @@ export default class StandardRouteModel extends BaseModel {
     }
 
     /**
+     * Returns an array of unique fix names used by any portion of the procedure
+     *
+     * @for StandardRouteModel
+     * @method getAllFixNamesInUse
+     * @return {array<string>} ['fixname', 'fixname', 'fixname', ...]
+     */
+    getAllFixNamesInUse() {
+        const allFixNames = [
```

Comment:
well all of them will at least return an empty array if it all goes wrong, and they are allowed to have some of them empty anyway. So there should be no throw from trying to do `...undefined` or something because it will always at least be `...[]`, which resolves to `[]`.